src/redfish/rest/v1.py

```diff
@@ -267,6 +267,32 @@ def session_location(self):
         self._session_location = self._http_response.getheader('location')
         return self._session_location
 
+    @property
+    def task_location(self):
+        """Return if we're a PATCH/POST in with a task link """
+        return self.session_location

```

Comment:
We should have a different property specific for tasks; even though they come back in the same header, the context is very different:
- "Location" for a Session happens when you POST to the Session Collection
- "Location" for a Task happens when you get back a 202